### PR TITLE
Account classification

### PR DESCRIPTION
**Pull request by NemTechCN**
Allow users to quickly find the corresponding account by network type
image

### CODE REVIEW

File: src/views/login/create-account/CreateAccountTs.ts [hunk 1]
```diff
@@ -14,7 +14,6 @@ export class CreateAccountTs extends Vue {
         hint: '',
         currentNetType: NetworkType.MIJIN_TEST,
     }
-    networkTypeList = networkTypeConfig
```
Comment (decentraliser) [inline src/views/login/create-account/CreateAccountTs.ts]: networkTypeList is referenced in CreateAccount.vue, but never set

**Comment by decentraliser** [comment]
please make sure this is OK before re-submitting: https://nem2.slack.com/archives/CME5PV744/p1571759179132700

File: src/views/login/create-account/CreateAccountTs.ts (hunk 1 above)
Comment (NemTechCN) [reply src/views/login/create-account/CreateAccountTs.ts]: fixed

**Comment by NemTechCN** [comment]
> please make sure this is OK before re-submitting: https://nem2.slack.com/archives/CME5PV744/p1571759179132700

fixed

**Pull request closed without merging**